source/assets/stylesheets/styles/_blog_article.scss

```diff
@@ -0,0 +1,279 @@
+.blog-header__container {
+  background-color: $header-background-color;
+  box-sizing: border-box;
+  padding: 100px 30px 30px;
+  width: 100%;
+}
+
+.blog-header__article-header {
+  @include breakpoint($blog-container-width) {
+    box-sizing: border-box;
+    margin: 0 auto;
+    max-width: $blog-container-width;
+    padding: 40px 0;
+  }
+}
+
+.blog-header__article-title {
+  font-size: 2.1em;
+  font-weight: bold;
+  line-height: 1.3em;
+  padding-bottom: 30px;
+
+  @include breakpoint($blog-container-width) {
+    padding-bottom: 50px;
+  }
+}
+
+.blog-header__article-author {
+  color: $author-name-color;
+  display: inline;
+  font-size: 1em;
+  letter-spacing: 0.02em;
+  padding-left: 2px;
+  text-decoration: none;
+}
+
+.blog-header__article-published-date {
+  color: $article-published-date;
+  display: inline;
+  font-size: 0.8em;
+
+  &:before {
+    content: "-";
+    padding-left: 5px;
+    padding-right: 5px;
+  }
+}
+
+.blog-header__article-tags {
+  display: none;
+
+  @include breakpoint($tablet-breakpoint) {
+    display: block;
+    padding-top: 40px;
+  }
+}
+
+.blog-header__article-tag-link {
+  @include breakpoint($tablet-breakpoint) {
+    display: inline-block;
+    padding-left: 15px;
+    padding-right: 15px;
+
+    a {
+      color: $medium-gray;
```

Comment:
All variables changed to describe colours. Although I'm thinking it'll be good to remove all `color` from names? They're quite long now, and it might be better to have:
`$comments-container-background` 
instead of:
`$comments-container-background-color`
especially that they're always used in context of colour properties. And our variables are color variable in majority so it's a bit repetitive.